src/modules/vtol_att_control/vtol_type.cpp

```diff
@@ -136,6 +136,9 @@ bool VtolType::init()
 		}
 	}
 
+	_spoiler_setpoint_with_slewrate.setSlewRate(1.f); //at least 1s from no spoiler deflection to full deflection
+	_flaps_setpoint_with_slewrate.setSlewRate(1.f); //at least 1s from no flaps deflection to full deflection
+
```

Comment:
same comment - can we make `1.f` named constexpr?

Reply:
https://github.com/PX4/PX4-Autopilot/pull/19329/commits/98ea2aeb1c80f716c5335c194fbe02116df85ca6